docs/changelog.rst

```diff
@@ -28,6 +28,8 @@ Bug fixes:
   ``r128_album_gain`` fields was changed from integer to float to fix loss of
   precision due to truncation.
   :bug:`4169`
+* :doc:`/plugins/convert`: The ``pretend`` and ``quiet`` flags are now respected
+  when ``delete_originals`` is enabled.
```

Comment:
Hmm, maybe I'm missing something, but I would have expected this to say "when `auto` is enabled" or "when automatically converting on import"? I think we're just changing what happens when doing the convert-on-import thing—regardless of `delete_originals`, if I'm not mistaken.

Reply:
Yes you're right, sorry. I'm conflating two different things here.

I originally started this PR because I saw `delete_originals` still print logs despite `quiet` being enabled. While fixing that I noticed that `pretend` was also not respected - as you point out, by the converter in general, regardless of `delete_originals` - so I figured I'd fix that too.

I'll make these two separate changelog entries to avoid the confusion.